.github/actions/setup-dependencies/windows/action.yml

```diff
@@ -45,8 +45,15 @@ runs:
           qt6-5compat:p
           qt6-networkauth:p
           cmark:p
+          tomlplusplus:p
           quazip-qt6:p
 
+    - name: List pacman packages (MinGW)
```

Comment:
Why do we need this step?

Reply:
Probably for debugging

Reply:
It pains me greatly when commit messages are not read https://github.com/PrismLauncher/PrismLauncher/pull/3757/commits/9d79695512acbfd487760400f11c68fd2dfa5a38 😢 (/j)